app/routes/dashboard.js

```diff
@@ -1,11 +1,31 @@
 import Ember from "ember";
 import AuthenticatedRouteMixin from 'simple-auth/mixins/authenticated-route-mixin';
 
-export default Ember.Route.extend(AuthenticatedRouteMixin, {
-	i18n: Ember.inject.service(),
+const { inject, Route, RSVP } = Ember;
+const { service } = inject;
+const { hash } = RSVP;
+
+export default Route.extend(AuthenticatedRouteMixin, {
+	i18n: service(),
+	currentUser: service(),
 	today: new Date(),
-	setupController: function(){
-    this._super.apply(arguments);
+
+	beforeModel() {
+		return this.get('currentUser.model');
+	},
+
+	model() {
+		const user = this.get('currentUser.user');
```

Comment:
OK - For now this should just be currentUser.model and then take out the set('user' stuff.  I will refactor this whole thing once we can update to ember-simple-auth 1.0 which is waiting on a few dependancies.  It will be much easier to manage the refactor if I can search for currentUser.model everywhere are replace it.